src/schemas/json/traefik-v2.json

```diff
@@ -1,435 +1,664 @@
+            },
+            "udp": {
+              "type": "object",
+              "properties": {
+                "timeout": {
+                  "type": "string"
+                }
+              }
             }
           }
         }
       }
     },
-    "provider-marathon": {
+    "experimental": {
       "type": "object",
       "properties": {
-        "constraints": {
-          "type": "string"
-        },
-        "trace": {
+        "kubernetesGateway": {
           "type": "boolean"
         },
-        "watch": {
+        "http3": {
           "type": "boolean"
         },
-        "endpoint": {
-          "type": "string"
-        },
-        "defaultRule": {
-          "type": "string"
-        },
-        "exposedByDefault": {
+        "hub": {
           "type": "boolean"
         },
-        "dcosToken": {
-          "type": "string"
-        },
-        "tls": {
+        "plugins": {
           "type": "object",
           "properties": {
-            "ca": {
-              "type": "string"
-            },
-            "caOptional": {
-              "type": "boolean"
-            },
-            "cert": {
-              "type": "string"
-            },
-            "key": {
-              "type": "string"
+            "Descriptor0": {
+              "type": "object",
+              "properties": {
+                "moduleName": {
+                  "type": "string"
+                },
+                "version": {
+                  "type": "string"
+                }
+              }
             },
-            "insecureSkipVerify": {
-              "type": "boolean"
-            }
-          }
-        },
-        "dialerTimeout": {
-          "type": "integer"
-        },
-        "responseHeaderTimeout": {
-          "type": "integer"
-        },
-        "tlsHandshakeTimeout": {
-          "type": "integer"
-        },
-        "keepAlive": {
-          "type": "integer"
-        },
-        "forceTaskHostname": {
-          "type": "boolean"
+            "Descriptor1": {
+              "type": "object",
+              "properties": {
+                "moduleName": {
+                  "type": "string"
+                },
+                "version": {
+                  "type": "string"
+                }
+              }
+            }
+          }
         },
-        "basic": {
+        "localPlugins": {
           "type": "object",
           "properties": {
-            "httpBasicAuthUser": {
-              "type": "string"
+            "Descriptor0": {
+              "type": "object",
+              "properties": {
+                "moduleName": {
+                  "type": "string"
+                }
+              }
             },
-            "httpBasicPassword": {
-              "type": "string"
+            "Descriptor1": {
+              "type": "object",
+              "properties": {
+                "moduleName": {
+                  "type": "string"
+                }
+              }
             }
```

Comment:
The plugins configuration is just a raw object/map, `Descriptor0` and `Descriptor1` are just examples.

You can see another (more realistic) example here: https://github.com/traefik/plugindemo

Reply:
@ldez thanks, missed that when generated schema from sample. Updated with pattern property definition